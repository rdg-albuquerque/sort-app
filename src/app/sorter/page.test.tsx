import Sorter from "@/app/sorter/page"
import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil"

describe("Sorter page", () => {
  test("Renders sorter page correctly", () => {
    const { container } = render(
      <RecoilRoot>
        <Sorter />
      </RecoilRoot>
    )

    expect(container).toMatchSnapshot()
  })
})
