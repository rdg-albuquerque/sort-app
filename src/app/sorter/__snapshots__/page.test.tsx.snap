// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sorter page Renders sorter page correctly 1`] = `
<div>
  <div
    class="flex min-h-screen flex-col items-center gap-24 p-24"
  >
    <h1>
      Quem vai tirar o papelzinho?
    </h1>
    <form>
      <select
        class="text-black"
        data-testid="select"
        id="participants-select"
        name="participants-select"
      >
        <option
          class="text-black"
          disabled=""
          hidden=""
          selected=""
          value="default"
        >
          Selecione seu nome
        </option>
        <option
          class="text-black"
          value="Rodrigo"
        >
          Rodrigo
        </option>
        <option
          class="text-black"
          value="João"
        >
          João
        </option>
        <option
          class="text-black"
          value="Maria"
        >
          Maria
        </option>
        <option
          class="text-black"
          value="Ana"
        >
          Ana
        </option>
      </select>
    </form>
    <p>
      Clique em sortear para ver quem é seu amigo secreto
    </p>
    <button
      class="bg-[#4646bb] text-white px-[40px] py-[18px] rounded-[38px]"
    >
      Sortear!
    </button>
    <a
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;
